iocore/net/quic/QUICPadder.cc

```diff
@@ -57,11 +57,7 @@ QUICPadder::will_generate_frame(QUICEncryptionLevel level, uint32_t seq_num)
   }
 
   this->_latest_seq_num = seq_num;
-  if (level == QUICEncryptionLevel::INITIAL) {
-    return true;
-  } else {
-    return this->_need_to_fire[static_cast<int>(level)] > 0;
-  }
+  return true;
```

Comment:
Should we change this to always add padding? I'm ok with it but I'm still not too sure about what cases we should pad packets.

Reply:
Yes we should. Otherwise the decryption will be failed by peer because of the bad sample length.  In original code it pad to minimus packet length (3 bytes) for every short packets.

Reply:
> bad sample length

Ah, I see. I forgot it. Thanks.